src/packages/core/components/section/section-sidebar-context-menu/section-sidebar-context-menu.element.ts

```diff
@@ -101,16 +101,25 @@ export class UmbSectionSidebarContextMenuElement extends UmbLitElement {
 			}
 			#action-modal {
 				position: absolute;
-				left: 300px;
+				left: var(--umb-section-sidebar-layout-width);
 				height: 100%;
 				z-index: 1;
 				top: 0;
-				width: 300px;
+				width: var(--umb-section-sidebar-layout-width);
 				border: none;
 				border-left: 1px solid var(--uui-color-border);
 				border-right: 1px solid var(--uui-color-border);
 				background-color: var(--uui-color-surface);
 			}
+
+			#action-modal h3 {
```

Comment:
Consider targeting a class rather than all h3 elements so people can opt into this styling

Reply:
Should not be a problem here cause of Shadow-DOM. I'll go ahead and get these corrections merged and then we can always revisit the h3 selector. :-)